includes/polling/wireless/canopy-generic.inc.php

```diff
@@ -290,4 +290,21 @@
         $graphs['canopy_generic_freq'] = true;
         unset($rrd_filename, $freq);
     }
+
+    $downlinkutilization = str_replace('"', "", snmp_get($device, "frUtlLowTotalDownlinkUtilization.0", "-Ovqn", "WHISP-APS-MIB"));
```

Comment:
yeah, just kept things the same as the existing code so as not to confuse someone who's looking at it later, The entire thing need to be rewritten to use a multiget, I was just trying to shoehorn my requirements in within the time window I had.

I'll see if I can find time to rewrite today.